plugins/BEdita/API/src/Controller/HistoryController.php

```diff
@@ -12,53 +12,35 @@
  */
 namespace BEdita\API\Controller;
 
-use BEdita\Core\Model\Action\ListEntitiesAction;
-use BEdita\Core\Utility\JsonApiSerializable;
-use Cake\Core\Configure;
-use Cake\ORM\TableRegistry;
+use Cake\ORM\Association;
 
 /**
  * Controller for `/history` endpoint.
  *
  * @since 4.1.0
+ * @property \BEdita\Core\Model\Table\HistoryTable $Table
  */
-class HistoryController extends AppController
+class HistoryController extends ResourcesController
 {
     /**
-     * History table
-     *
-     * @var \Cake\ORM\Table
+     * @inheritDoc
      */
-    protected $HistoryTable;
+    public $modelClass = 'History';
 
     /**
      * @inheritDoc
      */
-    public function initialize(): void
-    {
-        parent::initialize();
-
-        $historyTable = (string)Configure::read('History.table', 'History');
-        $this->HistoryTable = TableRegistry::getTableLocator()->get($historyTable);
-    }
+    protected $_defaultConfig = [
+        'allowedAssociations' => [
+            'user' => ['users'],
+        ],
+    ];
```

Comment:
```suggestion
     * @inheritDoc
     */
    protected $_defaultConfig = [
        'allowedAssociations' => [
            'user' => ['users'],
        ],
    ];

    /**
     * @inheritDoc
     */
    public function initialize(): void
    {
        $this->modelClass = (string)Configure::read('History.table', 'History');

        parent::initialize();
    }
```

Reply:
This way we keep the possibility to use a different table via `History.table` config.